python/doc/theory/reliability_sensitivity/form_approximation.rst

```diff
@@ -3,106 +3,75 @@
+nearest to the origin of the standard space. Thus, :math:`P^*` is the result of a constrained
+optimization problem.
+
+**Step 3**: In the standard space, approximate the limit state surface in the
+standard space by a linear surface at the design point :math:`P^*`. Then, the probability
+:math:`P_f` :eq:`PfU3` where the limit state surface has been approximated by a linear surface (hyperplane) can be obtained exactly, thanks to the rotation invariance of the standard distribution :math:`f_{\vect{U}}`::doc:
+
+.. math::
+    :label: PfFORM
 
         P_{f,FORM} =
             \left|
             \begin{array}{ll}
-              E(-\beta_{HL}) & \mbox{if the origin of the $\vect{u}$-lies in the domain $\cD_f$} \\
-              E(+\beta_{HL}) & \mbox{otherwise}
+              E(-\beta) & \mbox{if the origin fails}, \\
+              E(+\beta) & \mbox{otherwise}
             \end{array}
             \right.
 
-   | where :math:`\beta_{HL}` is the Hasofer-Lind reliability index,
-     defined as the distance of the design point
-     :math:`\vect{u}^*` to the origin of the standard space and
-     :math:`E` the marginal cumulative density function of the spherical
-     distributions in the standard space.
-   | Let us recall here that in the Rosenblatt standard
-     space, random vectors follow the standard normal distribution (with
-     zero mean, unit variance and unit correlation matrix), which
-     implies that :math:`E = \Phi`. In the Generalized Nataf standard
-     space, random vectors follow some spherical distributions, with
-     zero mean, unit variance, unit correlation matrix and which type
-     :math:`\psi` is the one of the copula of the physical random vector
-     :math:`\vect{X}` : in that case, :math:`E` is the 1D cumulative
-     distribution function with zero mean, unit variance and which type
-     is :math:`\psi`.
-
-Here, the event considered is explained directly from the limit state
-function :math:`g(\vect{X}\,,\,\vect{d})` : this is the classical
-structural reliability formulation.
-However, if the event is a threshold exceedance, it is useful to
-explicit the variable of interest
-:math:`Z=\tilde{g}(\vect{X}\,,\,\vect{d})`, evaluated from the model
-:math:`\tilde{g}(.)`. In that case, the event considered, associated to
-the threshold :math:`z_s` has the formulation:
-:math:`\cD_f = \{ \vect{X} \in \Rset^n \, / \, Z=\tilde{g}(\vect{X}\,,\,\vect{d}) > z_s \}`
-and the limit state function is :
-:math:`g(\vect{X}\,,\,\vect{d}) = z_s - Z = z_s - \tilde{g}(\vect{X}\,,\,\vect{d})`.
-:math:`P_f` is the threshold exceedance probability, defined as :
-:math:`P_f = P(Z \geq z_s) = \int_{g(\vect{X}\,,\,\vect{d}) \le 0}  \pdf\, d\vect{x}`.
+where :math:`\beta` is the Hasofer-Lind reliability index, defined as the distance of the design point
+:math:`P^*` to the origin of the standard space and :math:`E` the marginal cumulative density function of
```

Comment:
Not sure my suggestion is well formulated, but I think "the" marginal distribution might mislead readers. It is necessary IMO to emphasize that all marginals of the spherical distribution are the same.
```suggestion
:math:`P^*` to the origin of the standard space and :math:`E` the marginal cumulative density function along any direction of
```

Reply:
yes!